packages/web-runtime/src/App.vue

```diff
@@ -26,14 +26,10 @@
             :logo-img="logoImage"
             :product-name="productName"
             :nav-items="sidebarNavItems"
-            :hide-nav="sidebar.navigationHidden"
```

Comment:
Are we removing the general capability of hidding the sidebar? 🤔

Reply:
The whole sidebar can be still hidden with `isSidebarVisible`. This is only about the navigation items. And from my understanding the sidebar should behave now only as navigation so I do not see much point in giving the option to hide it :shrug: